### ISSUE TITLE
Site is completely blocked by AdBlockers (because of the analytics.js)

### ISSUE THREAD

**Issue by Sandtalker** [gdg-x/hoverboard#107]
Hi!

There is some serious problem with the site - it's almost completely blocked by ad blocking extensions, like the [uBlock](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm).

Konrad Przydział (konrad.przydzial@gmail.com) from Rzeszów, Poland has found out, that this can be fixed by changing the name of _scripts/analytics.js_ to _scripts/analytics-script.js_ :)

Website WITH uBlock:
![image](https://cloud.githubusercontent.com/assets/988380/17855655/1a7ffd2a-687a-11e6-9f2d-b21fce8e30d0.png)

Webstire WITHOUT uBlock:
![image](https://cloud.githubusercontent.com/assets/988380/17855661/25b45862-687a-11e6-97cd-4e509a254abf.png)

Greetz!

**Comment by ozasadnyy** [comment]
@Sandtalker, it was fixed in #105 
Duplicate of #102 and #104 .
